spec/models/car_spec.rb

```diff
@@ -0,0 +1,64 @@
+require 'rails_helper'
+
+RSpec.describe Car, type: :model do
+  # tests go here
+  before :each do
+    @car_first = Car.create(
+      name: 'Bettle',
+      photo: 'photo_link',
+      description: 'Nice yellow car',
+      price: 25_000,
+      reservation_fee: 250
```

Comment:
Here you added the "reservation_fee" field but it's not necessary anymore as we updated it to be in the form of a method inside the controller

Make sure you get the latest changes from the dev branch & update the tests 🚀

Reply:
Done! I have modified the tests to match the modifications in the controller